mito-ai/src/Extensions/AiChat/ChatMessage/ChatInput.tsx

```diff
@@ -310,6 +305,8 @@ const ChatInput: React.FC<ChatInputProps> = ({
                             title={context.type === 'db' && context.display ? context.display : context.value}
                             type={context.type}
                             onRemove={() => setAdditionalContext(additionalContext.filter((_, i) => i !== index))}
+                            notebookTracker={context.type === 'active_cell' ? notebookTracker : undefined}
+                            activeCellID={context.type === 'active_cell' ? activeCellID : undefined}
```

Comment:
I think for both of these it makes more sense to pass them through to the `SelectedContextContainer` regardless of if its of type active_cell or not. 

Then, we can let the `SelectedContextContainer` be in charge of the logic